templates/repo/issue/view_content/pull.tmpl

```diff
@@ -479,6 +477,10 @@
 					</button>
 				</div>
 			{{end}}
+
+			{{if and .ShowMergeInstructions .Issue.PullRequest.HeadRepo}}
```

Comment:
Why was the `.Issue.PullRequest.HeadRepo` previously not necessary?

Reply:
Before, there are various checks for `AllowMerge`, at that moment, `HeadRepo` can be sure to be non-nil.